### ISSUE TITLE
Layers >2 are not being created correctly

### ISSUE THREAD

**Issue by basepi** [kiibohd/KiiConf#16]
If I select any layers greater than the default 0-2, those layers are turned into JSON, but have no corresponding KLL files for those layers. Have tried both layer 3 and layer 7.

Could be related to #15 as not even the existing layers are being converted to KLL correctly.

Happy to help further diagnose the issue.

Using an ergodox infinity.

**Comment by basepi** [comment]
AHA! I figured it out.

The configurator doesn't write KLL for keys which are set to their default. In my case, I was trying to use layer 3 as an QWERTY override of my default layer, which is colemak. Because all keys in layer 3 were their default values, it didn't write out KLL for them. Thus, no layer 3 when I loaded up the firmware.

For now, I'm just writing and compiling my own KLL. But I feel like this could be a common use case. In my opinion, the configurator should write KLL for any key which is explicitly bound in the configurator, even with the default value. Much easier than conflict detection, and it allows for reference-KLLs to be generated (in case people want to write their own).
